### PR TITLE
Chef 14 post-migration fixes

### PR DESCRIPTION
**Pull request by detjensrobert**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -18,7 +18,6 @@
 depends          'osl-munin'
 depends          'osl-postfix'
 depends          'percona', '~> 0.16.1'
-depends          'sysctl'
 depends          'yum'
```
Comment (ramereth) [inline metadata.rb]: Actually, this should be removed as well.

**Comment by ramereth** [comment]
@detjensrobert looks like there are dependency issues with the latest razor-client gem:
``` console
$ /usr/bin/gem install razor-client --no-rdoc --no-ri -v "1.9.6"
ERROR:  Error installing razor-client:
        faraday requires Ruby version >= 2.3.
```
This is due to the fact that the system ruby that's installed is 2.0.0 and we won't be upgrading that. A workaround will be to set the version for the ``gem_package`` in the razor recipe to version ``1.9.1`` which seems to be the last version that is installable. A comment there would be good.

**Comment by detjensrobert** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.3.0.  Have a nice day!

**Pull request merged**